File: Java/Card.java
// Clase Tarjeta (Hereda de la clase pago)
class Card extends Payment {
    // Atributos o propiedades de la clase
    Integer id;
    Integer cardNumber;
    Integer CVV;
    String date;

    // Método constructor
    public Card(Integer cardNumber, Integer CVV, String date) {
        super();
        this.cardNumber = cardNumber;
        this.CVV = CVV;
        this.date = date;
    }
}

File: PHP/Card.php
<?php
//Importamos las clases
require_once('Payment.php');
//Clase Tarjeta (Hereda de la clase Pago)
class Card extends Payment
{
    //Atributos o propiedades de la clase
    public $id;
    public $numberCard;
    public $CVV;
    public $date;

    //Método constructor
    public function __construct($numberCard,$CVV,$date){
        $this->numberCard=$numberCard;
        $this->CVV=$CVV;
        $this->date=$date;
    }
}